.docker/source/Dockerfile

```diff
@@ -4,35 +4,30 @@
 FROM moveit/moveit:melodic-ci-shadow-fixed
 MAINTAINER Dave Coleman dave@picknik.ai
 
-ENV CATKIN_WS=/root/ws_moveit
-RUN mkdir -p $CATKIN_WS/src
-WORKDIR $CATKIN_WS/src
+# Commands are combined in single RUN statement with "apt/lists" folder removal to reduce image size
+# https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers
+RUN \
+    mkdir src && \
+    cd src && \
 
-# Download moveit source
-RUN wstool init . && \
+    # Download moveit source so that we can get necessary dependencies
+    wstool init . && \
     wstool merge https://raw.githubusercontent.com/ros-planning/moveit/${ROS_DISTRO}-devel/moveit.rosinstall && \
-    wstool update
+    wstool update && \
+
+    # Update apt package list as cache is cleared in previous container
+    # Usually upgrading involves a few packages only (if container builds became out-of-sync)
+    apt-get -qq update && \
+    apt-get -qq dist-upgrade && \
 
-# Update apt-get because osrf image clears this cache. download deps
-# Note that because we're building on top of melodic-ci, there should not be any deps installed
-# unless something has changed in the source code since the other container was made
-# (they are triggered together so should only be one-build out of sync)
-RUN apt-get -qq update && \
-    apt-get -qq install -y \
-        wget && \
     rosdep update && \
     rosdep install -y --from-paths . --ignore-src --rosdistro ${ROS_DISTRO} --as-root=apt:false && \
     rm -rf /var/lib/apt/lists/*
 
-# Replacing shell with bash for later docker build commands
-RUN mv /bin/sh /bin/sh-old && \
-    ln -s /bin/bash /bin/sh
-
 # Build repo
-WORKDIR $CATKIN_WS
-ENV TERM xterm
```

Comment:
this makes it easier to develop in a docker container:
https://andykdocs.de/development/Docker/Fixing+the+Docker+TERM+variable+issue